tracy-client-sys/Cargo.toml

```diff
@@ -43,6 +43,7 @@ ondemand = []
 manual-lifetime = ["delayed-init"]
 delayed-init = []
 callstack-inlines = []
+exit = []
```

Comment:
Fair enough, though that's definitely going to be annoying because now users need to disable default features. I wonder if `no-exit` does actually make more sense?

Reply:
I guess it could if one interprets it more like `hold-on-exit`, thus making it properly additive, rather than conceptually removing code that exits programs :)

Reply:
That works! I went with `flush-on-exit` because hold makes it sound like we'll wait forever.

Reply:
Mmmm, that makes CI ugly unfortunately.